server/entity/MediaRequest.ts

```diff
@@ -1104,11 +1110,9 @@ export class MediaRequest {
       switch (type) {
         case Notification.MEDIA_APPROVED:
           event = `${this.is4k ? '4K ' : ''}${mediaType} Request Approved`;
-          notifyAdmin = false;
```

Comment:
The `notifyAdmin` value is set to `false` here as this notification type doesn't pertain to admin users, but is of interest to the requesting user.